pkgs/servers/nfd/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, ndn-cxx
+, libpcap
+, fetchFromGitHub
+, boost
+, pkg-config
+, openssl
+, doxygen
+, wafHook
+, systemd
+, python3
+, python3Packages
+, withSystemd ? true
+, withWebSocket ? true
+}:
+let
+  pname = "nfd";
+  version = "0.7.1";
+  pythonPackages = p: with p; [ sphinx ];
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = lib.toUpper pname;
+    rev = "${lib.toUpper pname}-${version}";
+    sha256 = "1l9bchj8c68r6qw4vr1kc96jgxl0vpqa2vjkvy1xmhz92sivr6gi";
+    fetchSubmodules = withWebSocket;
```

Comment:
Please use the websocketpp from nixpkgs.

Reply:
Websocketpp is 0.8.1 in nixpkgs while this project uses 0.8.1-hotfix. The [changelist](https://github.com/cawka/websocketpp/tags) shows the hotfix is to support boost 17.

Reply:
I just noticed boost is an alias for boost169, and not boost175.
In the [release notes](https://named-data.net/doc/NFD/current/RELEASE_NOTES.html) the authors suggest at least 1.6.5.
What is the common practice in this case?

Reply:
I tested it with nixpkgs websocketpp and works ok